ppanggolin/formats/writeFlatPangenome.py

```diff
@@ -1143,7 +1143,12 @@ def write_pangenome_flat_files(pangenome: Pangenome, output: Path, cpu: int = 1,
         if regions:
             processes.append(p.apply_async(func=write_regions, args=(output, compress)))
         if borders:
-            processes.append(p.apply_async(func=write_borders, args=(output, dup_margin, compress)))
+            if pangenome.status["geneFamilySequences"] == "No":
```

Comment:
if the sequences are not their the other output of the write_borders function is not written. A good solution could be to split the function into two functions?